example/android/app/src/main/java/com/dataxad/flutter_mailer_example/MainActivity.java

```diff
@@ -0,0 +1,13 @@
+package com.dataxad.flutter_mailer_example;
+
+import androidx.annotation.NonNull;
+import io.flutter.embedding.android.FlutterActivity;
+import io.flutter.embedding.engine.FlutterEngine;
+import io.flutter.plugins.GeneratedPluginRegistrant;
+
+public class MainActivity extends FlutterActivity {
```

Comment:
Careful, this is the modern V2 FlutterActivity. When it's empty, you may also omit it, but instead add a EmbeddingV1Activity for testing compatibility with V1.

This repo has recently merged V1 + V2 support: https://github.com/nashfive/phone_number.